index.html

```diff
@@ -0,0 +1,182 @@
+          </div>
+          <div class="DefectProducts btOV-child">
+            <span class="iconify icon-btOverView" data-icon="bi:x"> </span>
+            <p>DEFECT PRODUCTS</p>
+            <p>43</p>
+          </div>
+        </div>
+
+        <!-- Inventory of the beer machine -->
+        <div class="inventory-st">
+          <div class="inventory-top">
+            <p>INVENTORY STATUS</p>
+          </div>
+          <div class="inventory-container">
+            <div class="BARLEY inventory-inner">
+              <div></div>
+              <div></div>
+              <div class="BARLEY inventory-overlay">
+                <p>BARLEY</p>
+                <p>0.100KG</p>
+              </div>
+            </div>
+            <div class="MALT inventory-inner">
+              <div></div>
+              <div></div>
+              <div class="MALT inventory-overlay">
+                <p>MALT</p>
+                <p>0.100KG</p>
+              </div>
+            </div>
+            <div class="HOPS inventory-inner">
+              <div></div>
+              <div></div>
+              <div class="HOPS inventory-overlay">
+                <p>HOPS</p>
+                <p>0.100KG</p>
+              </div>
+            </div>
+            <div class="WHEAT inventory-inner">
+              <div></div>
+              <div></div>
+              <div class="WHEAT inventory-overlay">
+                <p>WHEAT</p>
+                <p>0.100KG</p>
+              </div>
+            </div>
+            <div class="YEAST inventory-inner">
+              <div></div>
+              <div></div>
+              <div class="YEAST inventory-overlay">
+                <p>YEAST</p>
+                <p>0.100KG</p>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <!-- maintain info -->
+        <div class="maintains">
+          <div id="maintains-container">
+            <div></div>
+            <div></div>
+          </div>
+          <div id="maintains-bottom">
+            <p>100%</p>
+            <span class="iconify" data-icon="mdi:wrench"></span>
+          </div>
+        </div>
+
+        <!-- graph view -->
+        <div class="graphs">
+          <div class="graphs-left">
+            <p>50 PRODUCTS/MIN</p>
+          </div>
+          <div class="graphs-right">
+            <div class="graphs-top"><p>this is top graphs</p></div>
+            <canvas id="graphs"></canvas>
+          </div>
+        </div>
+
+        <!-- production control -->
+        <div class=production-control>
+          <div class="ptime">
+            <div>ALE PRODUCTION</div>
+            <div>00:11:52</div>
+          </div>
+          <div class="buttons">
+            <button id="stop">STOP</button>
+            <button id="clear">CLEAR</button>
+            <button id="start">START</button>
+            <button id="reset">RESET</button>
+          </div>
+          <div class="abort">
+            <button id="abort">ABORT</button>
```

Comment:
This button needs some kind of press feedback. The other buttons turn dark grey when pressed, maybe this could turn dark red?

Reply:
Yes that should not be that hard!